py-rattler/Cargo.toml

```diff
@@ -53,6 +48,7 @@ thiserror = "1.0.44"
 url = "2.4.1"
 
 openssl = { version = "0.10", optional = true }
+pep508_rs = "0.3.0"
```

Comment:
pep508_rs also exports python bindings I think, we could consider using those!

Reply:
Oh, let me have a look.